src/processes.c

```diff
@@ -2269,7 +2285,8 @@ static int ps_read (void)
 	}
 
 	/* Get the list of processes. */
-	procs = kvm_getprocs(kd, KERN_PROC_ALL, 0, sizeof(struct kinfo_proc), &count);
+	procs = kvm_getprocs(kd, KERN_PROC_ALL, 0, sizeof(struct kinfo_proc),
+                             &total_processes);
```

Comment:
Something's off with the indentation here.

Reply:
Yeah the tabs and spaces don't reconcile well between vim and the Github UI. Fixed.